.azure-pipelines.yml

```diff
@@ -33,3 +40,19 @@ jobs:
       - template: .ci/wasm-pack-test-headless.yml
         parameters:
           browser_flag: --chrome
+  - job: build_book
+    condition: |
+      eq(variables['Build.Reason'], 'PullRequest')
+    displayName: "Doc - build the book"
+    steps:
+      - template: .ci/install-mdbook.yml
+      - script: (cd guide && mdbook build)
+  - job: deploy_book
```

Comment:
why are build and deploy two separate jobs instead of just being steps that happen one after another?

Reply:
They don't get the same triggers.
On PR we only check if we can build the book.
On changes occurring on master we redeploy the book.

As I said, we could get rid of the first one if it's not useful.